sklearn/linear_model/logistic.py

```diff
@@ -17,22 +17,22 @@
 
 from .base import LinearClassifierMixin, SparseCoefMixin, BaseEstimator
 from .sag import sag_solver
+from ..exceptions import NotFittedError
+from ..externals import six
+from ..externals.joblib import Parallel, delayed
+from ..metrics import SCORERS
+from ..model_selection import check_cv
 from ..preprocessing import LabelEncoder, LabelBinarizer
 from ..svm.base import _fit_liblinear
 from ..utils import check_array, check_consistent_length, compute_class_weight
 from ..utils import check_random_state
 from ..utils.extmath import (logsumexp, log_logistic, safe_sparse_dot,
                              softmax, squared_norm)
 from ..utils.extmath import row_norms
-from ..utils.optimize import newton_cg
-from ..utils.validation import check_X_y
-from ..exceptions import NotFittedError
 from ..utils.fixes import expit
 from ..utils.multiclass import check_classification_targets
-from ..externals.joblib import Parallel, delayed
-from ..model_selection import check_cv
-from ..externals import six
-from ..metrics import SCORERS
+from ..utils.optimize import newton_cg
+from ..utils.validation import check_X_y
```

Comment:
I guessed my editor just put these automatically in alphabetic order, I can revert if necessary